convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -137,6 +137,8 @@ def run(self):
                     title="Outdated packages detected",
                     description="Please refer to the diagnosis for further information",
                     diagnosis=package_not_up_to_date_error_message,
+                    remediation="If you wish to ignore this message, set the environment variable "
+                    "'CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP' to 1.",
```

Comment:
Do we want to say something about disregarding/skipping/not performing the check instead of ignoring it? 🤔

Reply:
Using ignore is what we have across the other remediation fields for overriding a check, I wouldn't be opposed to changing it to something else for all of them though

Reply:
I'll make a jira for this change

Reply:
https://issues.redhat.com/browse/RHELC-1372